src/Console/Command/DescribeCommand.php

```diff
@@ -91,7 +91,7 @@ protected function configure(): void
     {
         $this->setDefinition(
             [
-                new InputArgument('name', InputArgument::REQUIRED, 'Name of rule / set.'),
+                new InputArgument('name', InputArgument::REQUIRED, 'Name of rule / set.', null, fn () => array_merge($this->getSetNames(), array_keys($this->getFixers()))),
```

Comment:
When using symfony/console < 6.1 you will only get completion for command names and option names, not for the argument/option values, because the  `$suggestedValues` param for InputArgument/InputOption didn't exist before.

Reply:
So especially when using php-cs-fixer via phar file, because the phar file uses `symfony/console` 5.4.

Also note that 5.4 supports `bash` only (`fish` came with 6.1, and `zsh` with 6.2).